player/command.c

```diff
@@ -8035,6 +8035,15 @@ void mp_option_run_callback(struct MPContext *mpctx, struct mp_option_callback *
         }
     }
 
+    if (opt_ptr == &opts->stream_bluray_opts->angle) {
```

Comment:
This seems to be null, see https://github.com/mpv-player/mpv/actions/runs/17533177291/job/49797178130

Reply:
Thanks, forgot an ifdef